src/memprof_js_library.js

```diff
@@ -0,0 +1,9 @@
+mergeInto(LibraryManager.library, {
+  emscripten_trace_record_allocation: function (address, size) {
+    memProf.mallocProxy(address, size);
+  },
+  emscripten_trace_record_free: function (address) {
```

Comment:
I think the convention is no space after keyword `function` and before opening parenthesis `(` here and other places in this PR.